Documentation/kernel.md

```diff
@@ -13,6 +13,7 @@ Default Kernel tree: 5.4
 | Raspberry Pi 4 | 4.19.127 |
 | Tinker Board | 5.4.63 |
 | Odroid-C2 | 5.7.19 |
+| Odroid-C4 (_experimental_) | 5.9.1 |
```

Comment:
```suggestion
| Odroid-C4 | 5.9.1 |
```
Please update also the script for kernel updates in `/scripts`

Reply:
Using this script updates the kernel version for all Odroid boards; is that what you intended?

Actually the other Odroid configs appear to have been updated to 5.9.1 already by commit 2b0fff31a367da05a8b6e171ab11860d4c7edee0; it's just the documentation files that are out of date.